t/01-hash.t

```diff
@@ -179,18 +179,7 @@ is_deeply(
     "slice_grep_map on keys",
 );
 
-is_deeply(
-    {
-        slice_grep_map { $_{$_} && $_{$_} > 1 } \%h,
-        (
-            a => "A",
-            b => "B",
-            c => "C"
-        )
-    },
-    {B => 2},
-    "slice_grep_map on values",
-);
+is_deeply({slice_grep_map { $_{$_} && $_{$_} > 1 } \%h}, {b => 2}, "slice_grep_map on values",);
```

Comment:
Der Test läuft in https://github.com/perl5-utils/Hash-MoreUtils/blob/master/lib/Hash/MoreUtils.pm#L166 und testet damit die falsche Funktion.